### ISSUE TITLE
Training metrics not reported in log file

### ISSUE THREAD

**Issue by albertoceballos** [Borda/kaggle_plant-pathology#9]
## 🐛 Bug

In the Pytorch Lightning Google Collab notebook the training loss, training accuracy, and training f1 score are not logged in the metrics file.
The training metrics are not logged by the Logger.
The validations metrics are logged in validation steps and are found in the log file.
The training metrics are not logged in the training step function and are not found in the log file.

How can these metrics be logged?

**Comment by edgarriba** [comment]
We found some issues with the logger in Lightning itself too. I believe is known issue, @Borda ?

**Comment by Borda** [comment]
I have open investigation over here - https://github.com/PyTorchLightning/pytorch-lightning/pull/6854
